utils/publish_all_packages.sh

```diff
@@ -48,9 +48,6 @@ fi
 
 NPM_PUBLISH_TAG="next"
 
-if [[ $1 == "--tip-of-tree" ]]; then
```

Comment:
`git status` check above fails if you update version beforehand.

Reply:
fixed